newrelic-weaver/src/main/java/com/newrelic/weave/utils/SynchronizedAnnotationNode.java

```diff
@@ -20,7 +20,7 @@
  * pretty strongly to an ASM version due to the fact that we had to override constructors in the original asm class. If
  * these constructors change or other accept() methods are added we will need to update accordingly.
  *
- * NOTE: If you are upgrading from org.ow2.asm:asm:8.0 / org.ow2.asm:asm-tree:8.0 make sure to double
+ * NOTE: If you are upgrading from org.ow2.asm:asm:9.2 / org.ow2.asm:asm-tree:9.2 make sure to double
```

Comment:
I checked out the ASM source code to `/tmp/asm` and used the following two commands to independently verify that no further changes to these classes are necessary:
1. A diff of all AWS classes for which New Relic ships with `Synchronized*` subclass:
   ```sh
    git grep -l 'If you are upgrading from' \
      | sed 's,.*Synchronized,asm-tree/src/main/java/org/objectweb/asm/tree/,' \
      | xargs git -C /tmp/asm diff ASM_8_0...ASM_9_2 --
    ```
2. A diff of all `org.objectweb.asm.tree` classes:
    ```sh
    git -C /tmp/asm diff ASM_8_0...ASM_9_2 -- asm-tree/src/main/java/org/objectweb/asm/tree
    ```